src/test/java/net/minecraftforge/debug/entity/living/ExtendedSpecialSpawnTest.java

```diff
@@ -1,5 +1,7 @@
 package net.minecraftforge.debug.entity.living;
```

Comment:
This file is missing the license header.

Reply:
Sorry, I didn't see the test had a license header. Commited the file with the header now.